base_layer/wallet/tests/transaction_service/service.rs

```diff
@@ -182,6 +182,7 @@ pub fn setup_transaction_service<T: TransactionBackend + Clone + 'static>(
                 refresh_neighbours_interval: Default::default(),
                 refresh_random_pool_interval: Default::default(),
                 random_peer_selection_ratio: 0.0,
+                useragent: format!("{}{}", "tari\\", env!("CARGO_PKG_VERSION")),
```

Comment:
Just a test so might as well be an empty string

Reply:
Done